src/common/components/Navbar/index.js

```diff
@@ -0,0 +1,45 @@
+import React, { cloneElement } from 'react';
+import PropTypes from 'prop-types';
+
+import ResponsiveNavigation from '../../../packages/responsive-navigation';
+
+import { Icon, Popover, Menu } from '../';
+
+const Navbar = ({ children }) => (
+  <ResponsiveNavigation>
+    {({ activePath, isMobile, hideNavigation, showNavigation, isNavigationVisible }) => {
+      const Menu = cloneElement(children, { activePath, isMobile });
```

Comment:
Shouldn't we use `React.Children.only(children)` here?

Or the `cloneElement` clones all the children while injecting the props to them?

Reply:
Good point i wanted to do it but then forgot